Pipfile

```diff
@@ -26,9 +26,14 @@ virtualenv = "*"
 
 [dev-packages]
 toml = "*"
+pytest = "*"
 pytest-timeout = "*"
 pytest-cov = "*"
-pytest = "*"
+hypothesis-auto = "*"
+hypothesis = "*"
+pytest-mypy = "*"
+mypy = "*"
+pre-commit = "==2.0.1"
```

Comment:
I would suggest leaving mypy and pre-commit on the testing environment - they are not direcly dependencies needed for the testsuite and we have container images in aicoe-ci that will test this.